DigitalLearningSolutions.Web/Controllers/LearningSolutions/SupportController.cs

```diff
@@ -0,0 +1,24 @@
+﻿namespace DigitalLearningSolutions.Web.Controllers.LearningSolutions
+{
+    using DigitalLearningSolutions.Web.Models.Enums;
+    using DigitalLearningSolutions.Web.ViewModels.LearningSolutions;
+    using Microsoft.AspNetCore.Mvc;
+
+    public class SupportController : Controller
+    {
+        [Route("/{application}/Support")]
+        public IActionResult Index(ApplicationType application)
+        {
+            if (ApplicationType.TrackingSystem == application ||
+                ApplicationType.Frameworks == application)
+            {
+                return View(
+                    "/Views/LearningSolutions/Support.cshtml",

```

Comment:
Could we fix this like we've done in TrackingSystem pages? We might actually not even want the controller to be in LearningSolutions folder given that we won't be including that in the url (could argue either way tbh, so feel free to do whatever is the most convenient 👍 )